src/commons/util/dummysigner.rs

```diff
@@ -0,0 +1,48 @@
+use rpki::repository::crypto::{KeyIdentifier, PublicKey, PublicKeyFormat, Signature, SignatureAlgorithm, Signer};
+
+use super::softsigner::SignerError;
+
+/// A dummy signer to prove that compilation with two different Signer implementations works
```

Comment:
That makes sense for now, but will this be removed when real signer implementations are added?

Reply:
Indeed. The other PR I am working (https://github.com/NLnetLabs/krill/pull/679) on already removes it (well, replaces it with a real KMIP signer).